p2p/protocol/protocol.go

```diff
@@ -93,7 +93,25 @@ func handleRequest(quaiMsg *pb.QuaiRequestMessage, stream network.Stream, node Q
 
 	switch decodedType.(type) {
 	case *types.Block:
```

Comment:
this commit should not have any changes to the block request handler. If there was a block handling bug, that bug should be addressed and reviewed independently of snap sync messaging